### PR TITLE
Removing codegen icg from pipeline.

### PR DESCRIPTION
**Pull request by karthijrk**
@ashwinstar @hardikar @tom-meyer Please take a look.

### CODE REVIEW

File: concourse/pipelines/pr_pipeline.yml [hunk 1]
```diff
@@ -90,19 +90,6 @@ jobs:
         timeout: 1h
         on_failure: *pr_failure
 
-      - task: icw_gporca_codegen_centos6
```
Comment (ashwinstar) [inline concourse/pipelines/pr_pipeline.yml]: We should replace running ORCA without codegen in PR pipeline. This currently will remove running ORCA from PR pipeline.

File: concourse/pipelines/pipeline.yml [hunk 2]
```diff
@@ -437,42 +437,6 @@ jobs:
...
-    - get: bin_gpdb
-      resource: bin_gpdb_centos6
-      passed: [compile_gpdb_centos6]
-      trigger: true
-    - get: centos-gpdb-dev-6
-  - task: ic_gpdb
-    file: gpdb_src/concourse/tasks/ic_gpdb.yml
-    image: centos-gpdb-dev-6
-    params:
-      MAKE_TEST_COMMAND: PGOPTIONS='-c optimizer=off -c codegen=on' installcheck-world
```
Comment (ashwinstar) [inline concourse/pipelines/pipeline.yml]: For other jobs where we explicitly say `codegen=off` should we loose them out as well to stop explicit calling of GUC ?

File: concourse/pipelines/pr_pipeline.yml (hunk 1 above)
Comment (karthijrk) [inline concourse/pipelines/pr_pipeline.yml]: oh!!. thanks for catching it. Not sure why it has codegen in its name.

Comment (karthijrk) [inline concourse/pipelines/pr_pipeline.yml]: Oh, We are running with codegen. Let me disable it.

File: concourse/pipelines/pipeline.yml (hunk 2 above)
Comment (karthijrk) [inline concourse/pipelines/pipeline.yml]: Just pushed that changes too.

File: concourse/pipelines/pr_pipeline.yml (hunk 1 above)
Comment (hardikar) [inline concourse/pipelines/pr_pipeline.yml]: icw_gporca_orca_centos6 sounds redundant. Looking at the pipeline, isn't there already a icw_gporca_centos6 job that runs with orca=on and codegen=off?

**Pull request merged**